yb_os_maint/yb_os_maint.py

```diff
@@ -444,22 +522,23 @@ def health_check(api_host, customer_uuid, universe, api_token, node):
             log('  All nodes, masters and t-servers are alive.')
 
         log('- Checking for master and tablet health')
-        if not node['isMaster']:
-            log('  ### Warning: node is not a Master ###')
-        if not node['isTserver']:
-            log('  ### Warning: node is not a TServer ###')
+        if node is not None:
```

Comment:
This should call the 'verify' subroutine, instead of repeating that logic here..